test/integration/patients/patient/archive.js

```diff
@@ -473,6 +474,24 @@ context('patient archive page', function() {
       .get('@wsHandleMessage')
       .should('have.been.calledOnce');
```

Comment:
Do we need this check still? Remove it?

Or we could move it down with the other one:

```js
cy
  .get('@wsHandleMessage')
  .should('have.been.calledOnce')
  .then(stub => {});
```